### DISCUSSION TITLE
How to move output .exe file in another directory outside builddir

### DISCUSSION THREAD

**Discussion by kirildi** [mesonbuild/meson#10832] [Q&A] [answered]
Hello guys. I'm kinda new to Meson(1 month or so usage). Can't find anything on this topic from few days.

On Windows project structure is like: 
ProjectDir
   | -- meson.build
   | -- includes/
   | -- src/
   | -- builddir/ - (meson builddir)
   | -- release/
.....
When compile the project with `meson -C builddir` builded `.exe` always goes inside `builddir`. I needed it to be in `release`.
Project is C++17 + GCC
End of `meson.build` is: 
```
executable('a',
           'main.cpp',
           include_directories : include,
           dependencies : libs,
           sources : src,
           install : true,
           install_dir: 'release')
```
I tried (as shown) to change output dir with `install_dir`, but doesn't have any effect. Thanks in advance.

**Comment by eli-schwartz** [comment]
install_dir is handled when you run `meson install`.

It sounds like what you want is similar to https://mesonbuild.com/Creating-OSX-packages.html just for Windows instead of macOS?

**Comment by xclaesse** [comment]
`meson install -C builddir --tags runtime --destdir ../release` maybe?

**Comment by kirildi** [comment, marked as answer]
Ok. I figured it out. I changed `prefix` and `bindir` from the `meson configure` like:

`meson configure -Dprefix='D:\ProjectDir' -Dbindir='release'`

then the  `meson compile` and `meson install` with the above `executable()` install parameters.

Could be good if it is somehow 1 command, but anyway... they can be combined in the VS Code task I guess. Thank you.
